src/utils/NebulaKeyUtils.cpp

```diff
@@ -51,7 +51,7 @@ std::string NebulaKeyUtils::edgeKey(size_t vIdLen,
        .append(srcId.data(), srcId.size())
        .append(vIdLen - srcId.size(), '\0')
        .append(reinterpret_cast<const char*>(&type), sizeof(EdgeType))
-       .append(reinterpret_cast<const char*>(&rank), sizeof(EdgeRanking))
+       .append(IndexKeyUtils::encodeInt64(rank))
```

Comment:
First of all, I think it is inappropriate to use the` IndexKeyUtils::encodeInt64` interface here

ps： If this is changed, a new interface is needed, right? Or this interface needs to be renamed

Reply:
How about use IndexKeyUtils::encodeValue?

Reply:
I think just copy the IndexKeyUtils::encodeInt64 and add a new function like NebulaKeyUtils::encodeRank?

Reply:
OK, will change it.